streams/src/main/java/org/apache/kafka/streams/processor/internals/ProcessorContextImpl.java

```diff
@@ -60,7 +60,7 @@ public RecordCollector recordCollector() {
     @Override
     public StateStore getStateStore(final String name) {
         if (currentNode() == null) {
-            throw new org.apache.kafka.streams.errors.TopologyBuilderException("Accessing from an unknown node");
+            throw new org.apache.kafka.streams.errors.StreamsException("Accessing from an unknown node");
```

Comment:
We can import `StreamsException` an get rid of the package prefix.

(We just did not import `TopologyBuilderException ` because it's a deprecated class and for imports we cannot suppress warnings about usage of deprecated code.

Btw: you can also remove `@SuppressWarning` from this method.